test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/integration-test/run-test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Trust the self-signed certificated used by the collector
+cp /cfg/otel-collector.crt /usr/local/share/ca-certificates/
+update-ca-certificates --verbose
+
+dotnet vstest OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests.dll --TestCaseFilter:CategoryName=CollectorIntegrationTests --logger:console;verbosity=detailed
```

Comment:
Why use `dotnet vstest` instead of `dotnet test`?

Reply:
Is it possible to display the detailed test execution output similar to the regular build workflow, summarizing the total number of tests ran, passed, and failed along with the names of the tests that were run?

https://github.com/open-telemetry/opentelemetry-dotnet/actions/runs/5720871835/job/15501540786?pr=4732

Reply:
> Why use dotnet vstest instead of dotnet test?

🤷 This test was previously being run using vstest and I just preserved that. Changed it to use dotnet test and seems to work fine.

> Is it possible to display the detailed test execution output similar to the regular build workflow, summarizing the total number of tests ran, passed, and failed along with the names of the tests that were run?

Yup. Fixed.